markdown/m2h/handlers/index.ts

```diff
@@ -86,18 +108,44 @@ export function buildLocalizedHandlers(locale: string): Handlers {
     blockquote(state: State, node: any): ReturnType<Handler> {
       const type = getNotecardType(node, locale);
       if (type) {
-        const isCallout = type == "callout";
-        if (isCallout) {
-          if (node.children[0].children.length <= 1) {
-            node.children.splice(0, 1);
-          } else {
-            node.children[0].children.splice(0, 1);
+        const isCallout = type.type == "callout";
+
+        if (type.isGFM) {
+          // Handle GFM proposed syntax
+          node.children[0].children[0].value =
+            node.children[0].children[0].value.replace(/\[!\w+\]\n/, "");
+
+          // If the type isn't a callout, add the magic keyword
+          if (!isCallout) {
+            node.children[0].children.unshift({
+              type: "strong",
+              children: [
+                {
+                  type: "text",
+                  value: type.magicKeyword,
+                },
+              ],
+            });
+            node.children[0].children[1].value =
+              " " + node.children[0].children[1].value;
```

Comment:
Leave a comment for the localization (l10n-zh). Actually, we will not add spaces between the full-width punctuation (`：`) and content.

So it would be better to check if the locale if `zh-CN` or `zh-TW`, if so, the space may not to be added.

---

### Example

So, if we have the following markdown:

```md
> [!NOTE]
> 你好，世界！
```

We'd better show it in the following format:

```html
<p><strong>备注：</strong>你好，世界！</p>
```

There is no spaces between the prefix ("备注：") and the actual content ("一些需要引起读者注意的内容").